pkg/apis/pipeline/v1beta1/param_types.go

```diff
@@ -121,7 +121,7 @@ func (ps Params) ExtractNames() sets.String {
 	return names
 }
 
-func (ps Params) extractValues() []string {
+func (ps Params) ExtractValues() []string {
```

Comment:
Do we need docstring for this since it is changed to public? 🤔 But why the lint is not failing maybe we don't need? Or do we need it to be public? I tried to search the references but only found it used in v1beta1, maybe I missed?